command/watch.go

```diff
@@ -182,11 +185,19 @@ func (c *WatchCommand) Run(args []string) int {
 			cmd.Stdout = os.Stdout
 			cmd.Stderr = os.Stderr
 
+			// Watch for a stop
+			go func() {
+				<-wpStoppingCh
```

Comment:
Hey @ryanuber thanks for reviewing my PR!

Your suggestion is what I initially tried, but it did not work as desired because in Go it seems if a value is placed into a channel (e.g. `c.ShutdownCh`) and there are two receivers waiting for a value (e.g. here and on line 210), only one of the receivers will receive that value.